Components/alert.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
You should use meaningful `titles` for all HTML pages, it improves SEO, and reflects it the browser tab for easy accessibility.

For eg. here you can use the title as `Flame UI | Alert`